src/lib/autoPresets.js

```diff
@@ -671,5 +671,224 @@ vec2 get_velocity(vec2 p) {
   return v;
 }`,
     "particleCount": 30000
+  },
+  {
+    "name": "README 1",
+    "timeStep": 0.007,
+    "fadeOut": 0.998,
+    "dropProbability": 0.009,
+    "colorMode": 1,
+    "cx": -1.275949999999999,
+    "cy": -1.6277,
+    "w": 30.2937,
+    "h": 30.2937,
+    "code": `v.x = length(p)*min(sin(p.y),cos(p.x));
```

Comment:
Does this work on your end? I see errors saying that `v` is not defined. 

This probably needs to be wrapped into function explicitly (like other fields)...

Reply:
Ah, I see! Fixed :)